core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1011,6 +1013,30 @@ private[spark] object Utils extends Logging {
   }
 
   /**
+   * Convert a time parameter such as (50s, 100ms, or 250us) to microseconds for internal use. If
+   * no suffix is provided, the passed number is assumed to be in us.
+   */
+  def timeStringAsUs(str: String): Long = {
+    JavaUtils.parseTimeString(str, TimeUnit.MICROSECONDS)
```

Comment:
nit: if you call the corresponding method in `JavaUtils`, then `parseTimeString` can be private and the code here becomes a little simpler.